worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/util/FoliaTaskManager.java

```diff
@@ -34,23 +34,35 @@ public class FoliaTaskManager extends TaskManager {
 
     @Override
     public int repeat(@NotNull final Runnable runnable, final int interval) {
-        return fail();
+        Bukkit.getGlobalRegionScheduler().runAtFixedRate(
```

Comment:
This is a workaround - but it doesn't solve the issues where this method is called.
There basically is no place where we can still use repeat this way safely, so it should fail hard.

That also applies to the other methods.

Reply:
Yeah but we can’t use also the background thing from you. I tried by my self to use that. But the fawe runnables are never executed...

Reply:
That's why we need to change the logic, but the global scheduler is not meant to run arbitrary tasks, it's meant to run tasks that depend on the global state

Reply:
Let's discuss that on discord. I am more productive in german as english